index.html

```diff
@@ -1,26 +1,29 @@
-<!Doctype html>
-<html id ="main">
-    <head>
-    <link href="/style/style.css" type="text/css" rel="stylesheet">
-        <title id="title">Dr AndrewTufoin</title>
-    </head>
-    <body>
-        <h2>
-            MICROVERSE 
-        </h2>
-    <div>
-        <h4 id="tribute-info">
-            <p>
-                The tribute page lays emphasis on how microverse assist student who have the zeal<br>
-                and no knowledge about programming to become a remote software developer.
-            </p>
-    </h4>
-    <div id ="img-div" width="300">
-        <img id ="image">
-        <img src ="/images/mic-img.png" alt="microverse students" class="center">
-        <p id = "img-caption"><strong>An image of microverse students in class</strong></p>
-    </div>
-    </div>
-    <a id ="tribute-link" href="https://www.microverse.org/" target="-blank">visit microverse</a>
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title id="title">Dr AndrewTufoin</title>
+  </head>
+  <body>
+    <div class="full-body">
+      <h2>
+        MICROVERSE
+      </h2>
+      <div>
+        <h4>
+            The tribute page lays emphasis on how microverse assist student who have the zeal<br>
+            and no knowledge about programming to become a remote software developer.
```

Comment:
Good work, 😄  I think an introduction will have been necessary, microverse needs to be introduced at least don't you think 💯